### PR TITLE
pageObject for login feature

### PR DESCRIPTION
**Pull request by susmitathapa**
### Description
-This PR is related to implementation of page object model(POM) in login feature
-A new folder pageObjects is created

### CODE REVIEW

**Comment by kiranadh1452** [comment]
@susmitathapa 
This PR has unwanted file changes (there are changes in contact feature file and context which should not be the case). Fix that.

**Comment by kiranadh1452** [comment]
@susmitathapa 
This PR has unwanted file changes (there are changes in contact feature file and context which should not be the case). Fix that.

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]
This PR has unwanted file changes (there are changes in contact feature file and context which should not be the case). Fix that.

File: Features5/tests/pageObjects/loginPage.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//concept/implementation of page object model(POM)
+//loginpage page object
+
+class LoginPage {
+  constructor() {
+    //locators
```
Comment (SagarGi) [inline Features5/tests/pageObjects/loginPage.js]:
```suggestion
    //selectors
```

File: Features5/tests/pageObjects/loginPage.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    this.emailSelector = "input[name='email']";
+    this.passwordSelector = "input[name='password']";
+    this.loginSelector = "input[value='Login']";
+    this.logoutSelector = "span[class='hide-sm']";
+    this.popoverSelector = "div[class='alert alert-danger']";
+  }
+
+  async navigate() {
+    //navigation
+    await page.goto("http://localhost:3000");
```
Comment (SagarGi) [inline Features5/tests/pageObjects/loginPage.js]: put it to constructor

File: Features5/tests/pageObjects/registrationPage.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    this.navRegisterSelector = "div [href='/register']";
+    this.registerSelector = "input[value='Register']";
+    this.loginSelector = "input[value='Login']";
+    this.logoutSelector = "span[class='hide-sm']";
+    this.popUpSelector = "div[class='alert alert-danger']";
+  }
+
+  async navigateRegistrationPage() {
+    //navigation
+    await page.goto("http://localhost:3000");
```
Comment (SagarGi) [inline Features5/tests/pageObjects/registrationPage.js]: .

File: Features5/tests/stepDefinitions/userRegistrationContext.js [hunk 4]
```diff
@@ -1,73 +1,53 @@
...
   console.log("navigated to registration page");
-  await page.goto("http://localhost:3000");
-  await page.click(navRegisterSelector);
+  await registrationPage.navigateRegistrationPage();
 });
 
 When(
   "the user registers with following credentials:",
   async function (dataTable) {
     console.log("enters valid credentials");
```
Comment (SagarGi) [inline Features5/tests/stepDefinitions/userRegistrationContext.js]: remove this

**Review by SagarGi — APPROVED** [review]

**Pull request merged**